connector_dns_dnspod/readme.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Connector Dns Dnspod
+=====================
+
+This module aims to allows you to manage your DNSPod domain through Odoo.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (connector_dns)
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * No specific configuration needed.
+
+Usage
+=====
+
+To use this module, you need to:
+1.Create a backend which links to your dnspod.cn.
+2.When you create a domain belongs to the backend,if the domain 
+  export to the dnspod.cn successfully,the state will change to 
+  done,else exception.
+3.Record can be created only in the domain which state is done. 
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
```

Comment:
This connector is only compatible with dnspod.cn and not dnspod.com which has a different API.